tests/ecosystem/deployment/test_ocs_basic_install.py

```diff
@@ -0,0 +1,214 @@
+import json
```

Comment:
FYI this is basically old test case just changed few stuff which moved to pytest. So pleas don't look on code improvements here which were already implemented like this in [origin test](https://github.com/red-hat-storage/ocs-ci/blob/master/tests/test_ocs_basic_install.py)